globus_automate_client/queues_client.py

```diff
@@ -130,17 +133,17 @@ def receive_messages(
         params: Dict[str, Any] = {"max_messages": max_messages}
         if receive_request_attempt_id is not None:
             params["receive_request_attempt_id"] = receive_request_attempt_id
-        return self.get(f"/queues/{queue_id}/messages", params=params)
+        return self.get(f"/queues/{queue_id}/messages", query_params=params)
 
     def delete_messages(
         self, queue_id: str, receipt_handles: List[str]
     ) -> GlobusHTTPResponse:
         self.authorizer = get_authorizer_for_scope(QUEUES_RECEIVE_SCOPE)
         body = {"data": [{"receipt_handle": rh} for rh in receipt_handles]}
-        return self._request(
+        return self.request(
```

Comment:
Would it be more consistent to use the client's delete method? Based on how the rest of it looks we could do `self.delete(f"/queues/{queue_id}/messages", data=body)`

Reply:
The client's delete method doesn't have a *data* parameter.